src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-between-cells;
+
+  width: $calendar-width;
+
+  &__square {
+    @include square(98px);
```

Comment:
Better store cell size as a global variable. Why 98 px? Set proper `box-sizing` for all elements.